src/transforms/transform_chips.stories.mdx

```diff
@@ -0,0 +1,27 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import { transform_chips } from "./column_transforms";
+import { TransformExample } from "./utils/TransformExample";
+
+<Meta title="Utils/Transforms/Chips"/>
+
+# transform_chips
+
+Convert an array of MosaicLabelValue elements into Chips.
```

Comment:
Just a spelling suggestion, conjugate the convert verb in the third person like the following:

**Converts** an array of MosaicLabelValue elements into Chips.